dev-java/fec/fec-1.0.4-r5.ebuild

```diff
@@ -0,0 +1,60 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+# There seems to be unit tests, but they are in such a state.
```

Comment:
please be more specific here, it's too abstract to me. what is exactly the issue with the tests?